runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1355,3 +1355,111 @@ TR_ResolvedJ9JITaaSServerMethod::getCachedResolvedMethod(TR_ResolvedMethodKey ke
       }
    return false;
    }
+
+bool
+TR_ResolvedRelocatableJ9JITaaSServerMethod::storeValidationRecordIfNecessary(TR::Compilation * comp, J9ConstantPool *constantPool, int32_t cpIndex, TR_ExternalRelocationTargetKind reloKind, J9Method *ramMethod, J9Class *definingClass)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *) comp->fe();
+
+   bool storeClassInfo = true;
+   bool fieldInfoCanBeUsed = false;
+   TR_AOTStats *aotStats = ((TR_JitPrivateConfig *)fej9->_jitConfig->privateConfig)->aotStats;
+   bool isStatic = false;
+
+   TR::CompilationInfo *compInfo = TR::CompilationInfo::get(fej9->_jitConfig);
+   TR_RelocationRuntime *reloRuntime = compInfo->getCompInfoForThread(fej9->vmThread())->reloRuntime();
```

Comment:
We should make sure that the reloRuntime stored in compInfo is of type TR_SharedCacheJITaaSRelocationRuntime

Reply:
Yeah, but that should probably go in a different commit, the one that implements RelocationRuntime.